pkg/operator/controllers/checker/ingresscert.go

```diff
@@ -0,0 +1,74 @@
+package checker
+
```

Comment:
Hi @hawkowl thank you for the check. Would you please summarize what this checker is doing in top comment.
A sentence is more than enough. Thank you, please think of it as a note for future you to help you remember fast.